### ISSUE TITLE
Error with the sign up

### ISSUE THREAD

**Issue by yadercenteno** [sapienza/CodeIgniter-admin-panel#9]
Hi sapiensa:

When I try to sign up, I get the 404 Not Found page, with the following message:
"Not Found

The requested URL /CodeIgniter-admin-panel/admin/signup was not found on this server.
Apache/2.2.22 (Ubuntu) Server at localhost Port 80"

I have configured the config.php and dababase.php files, as you mention on your Readme file, but nothing works.

What should I do?  

Help me, please.  This example looks very awsome for learn from it.

Regards,

Yader Centeno
León, Nicaragua

**Comment by yadercenteno** [comment]
Yes, I did it.  I configured $ config['base_url'] = 'http://localhost/CodeIgniter-admin-panel/'.

I install it on my localhost server in my laptop.   

What else can I do?

**Comment by MrLunie** [comment]
error came from the url . http://CodeIgniter-admin-panel-master/
everytime hitting submit or do something ,  the url change into CodeIgniter-admin-panel/admin/signup
the word master is missing , how can i fix this ? thanks in advance

**Comment by MrLunie** [comment]
OH ? em sorry for having populate this thread . :) the problem is just the $config['base_url']  = ""; in config file.. then , maybe , this thread must be closed .

**Comment by syscoid** [comment]
after already create and login user & password like this:
URL /CodeIgniter-admin-panel/admin/login/validate_credentials was not found on this server.
why?

**Comment by arditya** [comment]
i have like a trouble,, when i click "Login"  URL; http://localhost/adminpanelci/user/validate_credentials
404 Page Not Found The page you requested was not found.
why?

**Comment by chandan830401** [comment]
# [htaccess.txt](https://github.com/sapienza/CodeIgniter-admin-panel/files/260854/htaccess.txt)

**"The requested URL /CodeIgniter-admin-panel/admin/signup was not found on this server.

# Apache/2.2.22 (Ubuntu) Server at localhost Port 80"**

This issue has been solved. It is coming for .htaccess file.
In root directory there is missing .htaccess file. So if you create a .htaccess file with this content ----

RewriteEngine on
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule .\* index.php/$0 [PT,L] 

Then it should be run / working fine or successfully.
